src/components/GenderViolenceMain/__snapshots__/index.test.jsx.snap

```diff
@@ -21,11 +21,16 @@ exports[`renders correctly and matches the snapshot 1`] = `
       <p>
         Safe Place provides access to immediate help and support for victims of violence. We offer a shelter, psychological help, lawyers and doctors.
       </p>
-      <button
-        type="button"
+      <a
```

Comment:
if you ask some people this isn't valid html5 but i'm not requesting a change here, you can google "button in a tag" to learn more

Reply:
do you mean line 26 in src/components/GenderViolenceMain/index.jsx?

Reply:
i'm just reading the snapshot to see what the final html looks like, i don't actually know what the source of this change is but the snapshot is the source of truth of what html is showing up on the screen

but like i said you can leave as is, i don't think it matters as long as it works here

Reply:
okay, thank you